protos/feast/core/CoreService.proto

```diff
@@ -215,4 +220,39 @@ message ListProjectsRequest {
 message ListProjectsResponse {
     // List of project names (archived projects are filtered out)
     repeated string projects = 1;
-}
\ No newline at end of file
+}
+
+message GetFeatureStatisticsRequest {
+    // Optional filter which filters returned statistics by selected features. These
+    // features must be present in the data that is being processed.
+    repeated string feature_refs = 1;
```

Comment:
@zhilingc just a question. I know I fleshed out this API on the RFC, but we didn't actually talk about the implications of `feature_refs` vs a single `feature_set`. If we allow multiple feature references then that will be another form of division in the returned statistics if we are following our precalculation strategy. The return results will still be coming back in the form of

- feature set 1
    - dataset 1 stats
    - dataset 2 stats
- feature set 2
    - dataset 3 stats
    - dataset 4 stats

Is this what you were thinking as well?

Reply:
No, I dont see why we can't merge results across featuresets

Reply:
I think we can (at the feature column level), but I am more worried about statistics not lining up correctly. In the above example it is possible for dataset 1 & 2 to not line up with dataset 3 and 4 (different days, entities). Which means that individual statistics on feature sets could look amazing, and if we return them together in an aggregated form it might be misleading to users because they think this is what they can expect from their training dataset.

But when the join happens to produce the training dataset it could result in a bunch of missing values, despite the statistics showing a a high unique count for entities and a high feature value count.

Reply:
So I think its super important to be clear on what we are going to return here.

Perhaps aggregation should be a flag?

Reply:
As long as we commit to returning statistics at a `feature` level and not `feature set` level I think it shouldn't be our responsiblity to ensure that features across feature sets and datasets line up. 

I personally think its fair that the statistics returned are purely for the feature values in the store, not what it looks like when joined to other data. If users want statistics of that nature, they should be retrieving batch datasets from Feast serving and running TFDV on that data.